### ISSUE TITLE
Project cannot build locally on fresh clone due to missing sonatype properties

### ISSUE THREAD

**Issue by rwdalpe** [ndw/xmlcalabash1-gradle#5]
I'm opening this as requested in #3 . Maybe I'm just doing something completely wrong here, of course.

*Steps to reproduce*:

1. Freshly clone the project
2. Attempt to do anything involving gradle, even `gradle tasks`

You'll get an error 

```
FAILURE: Build failed with an exception.

* Where:
Build file '/tmp/xmlcalabash1-gradle/build.gradle' line: 126

* What went wrong:
A problem occurred evaluating root project 'xmlcalabash1-gradle'.
> Could not get unknown property 'sonatypeUsername' for object of type org.gradle.api.publication.maven.internal.deployer.DefaultGroovyMavenDeployer.
```

![xmlcalabashgradle_build_issue](https://user-images.githubusercontent.com/824357/36948102-66a526a8-1f8a-11e8-9de8-c92dbf1dd1e0.gif)

**Comment by liamquin** [comment]
Yes, i see this too. Deleting the stuff about sona from build.gradle makes that error go away and then you can get to failing onthe javadoc stage. But that's just a warning, it builds.
